erts/emulator/beam/jit/beam_jit_common.hpp

```diff
@@ -235,6 +235,33 @@ class Span {
     }
 };
 
+static const int BSC_OP_BINARY = 0;
```

Comment:
I'd prefer using enums so we'll get compiler warnings if we accidentally leave a case out:
```suggestion
typedef enum : Uint {
    BSC_OP_BINARY = 0,
    ...
    BSC_OP_LAST = ...
    BSC_OP_MASK = ...
} JitBSCOp;
```